ajax/libs/fontawesome-iconpicker/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "fontawesome-iconpicker",
+  "version": "1.3.1",
+  "description": "Font Awesome Icon Picker plugin for Twitter Bootstrap",
+  "homepage": "https://itsjavi.com/fontawesome-iconpicker/",
```

Comment:
The current link show 404 not found.
Let's use `https://farbelous.github.io/fontawesome-iconpicker/` as the homepage, what do you think?

Thanks.